src/pages/docs/projects/variables/tenant-variables.mdx

```diff
@@ -0,0 +1,117 @@
+
+There are two types of variable templates:
+
+- [Project templates](#project-templates) are variable values that differ between projects and environments for a tenant. For example, server names or database connection settings.
+- [Common templates](#adding-a-variable-template) are variable values that are common across all tenants but need a unique value per tenant. For example, website names, titles, headers, images, logo, URLs, contact information. These values don't change across projects and environments for a tenant.
+
+:::div{.hint}
+View a working example on our [samples instance](https://oc.to/samples-tenant-variables).
+:::
+
+## Project templates \{#project-templates}
+
+Project templates allow you to specify variables that can have different values per tenant/environment combination. A perfect example would be a connection string or a database server. With project templates, you define them at the project level.
+
+:::div{.hint}
+From **Octopus 2024.1**, we've improved the multi-tenancy variable management experience. For versions older than 2024.1, see [prior to 2024.1](#prior-to-2024-1)
+:::
+
+To add a project template:
+1. Navigate to the **Tenant Variables** page in any project.
+2. Click **Add project template**.
+3. Add the details to your template and click **Add**.
+
+|      | Description                              | Example |
+| ---- | ---------------------------------------- | ------- |
+| **Name** | The name of the variable template. This will also be the name given to the resulting variable value you can use in your deployment process. | `Tenant.Alias` |
+| **Label**         | The label that will be displayed when prompting for the variable value. | Tenant alias                             |
+| **Help text**     | The descriptive help text that will be displayed to provide the user with enough information to accurately provide the value. | A shortened, URL friendly, version of the tenant's name. |
+| **Control type** | You can select one of several different data types. This controls the user interface provided to collect the variable value, and determines how the variable value is interpreted. Note the variable values will be stored and interpreted as text. | Single-line text box, multi-line text box, drop down, checkbox, sensitive/password box, Azure account |
+| **Default value** | The value that will be given to the variable if an actual value is not provided. The default value can contain [variable binding expressions](/docs/projects/variables/variable-substitutions). | `https://#{Tenant.Alias}.myapp.com`      |
+| **Options** | (Only applies when control type is drop down). This defines the list of options available for the user to select from the drop down list. Enter each option on a new line. Use `|` to separate values and display text. | `Value1|Display text 1` <br>`Value2|Display text 2`  |
```

Comment:
> Use `|` to separate values

Looks like this needs to be escaped
<img width="666" alt="image" src="https://github.com/OctopusDeploy/docs/assets/108677358/06ac1720-8f44-43bf-bc44-72136bb6912e">